hugegraph-core/src/main/java/com/baidu/hugegraph/schema/PropertyKey.java

```diff
@@ -196,12 +199,13 @@ public String clazz() {
                 valid = this.checkDataType(value);
                 break;
             case SET:
-                valid = value instanceof Set;
-                valid = valid && this.checkDataType((Set<?>) value);
+                // support list and single, all covert 2 set
+                valid = this.checkDataType(toSet(value));
```

Comment:
keep the origin code "valid = valid && this.checkDataType((Set<?>) value)", just call toSet() before call check as needed.
we should keep the common code clearly.